services-js/commissions-app/src/client/CommissionsListSection.tsx

```diff
@@ -99,7 +110,7 @@ export default class CommissionsListSection extends React.Component<Props> {
           />
         </CollapsibleSection>
 
-        {commissionsSelectionError(this.props.errors)}
+        {commissionsSelectionError(errors, selectedCommissionIds.length > 0)}
```

Comment:
shouldn’t this be `selectedCommissionIds.length === 0`?

Reply:
No, we want to hide the error when length is 0, not show it. The idea is that this isn't a useful error to show, especially because we don't currently have blur handling.